src/styles/variables.css

```diff
@@ -10,7 +11,13 @@
 	--rounded-corners: 2px;
 	--short-animation-duration: .2s;
 
+	--block-spacing: 25px;
+	--line-spacing: 5px;
+
 	--title-bar-bg: var(--dojo-blue);
 	--title-bar-color: var(--dojo-white);
 	--underlay-bg: var(--dojo-shade-gray);
+
+	--focus-highlight: 0 0 4px 1px var(--dojo-light-blue);
```

Comment:
this looks like it's being used as a `shadow` and we already have a `subtle-shadow` variable so can we align these please and call this `--input-focus-shadow` ?